src/components/CreateResolution/CompleteResolutionSummary.vue

```diff
@@ -0,0 +1,180 @@
+        <v-col md="11" id="file-name-col">
+          <span>{{ getCreateResolutionResource.confirmSection.reviewSummaryText }}</span>
+        </v-col>
+      </v-row>
+    </section>
+  </v-card>
+</template>
+
+<script lang="ts">
+// Libraries
+import { Component, Mixins } from 'vue-property-decorator'
+import { Getter } from 'vuex-class'
+import { DateMixin } from '@/mixins'
+
+// Enums
+import { RouteNames } from '@/enums'
+import { CreateResolutionIF, CreateResolutionResourceIF } from '@/interfaces'
+
+@Component({})
+export default class CompleteResolutionSummary extends Mixins(DateMixin) {
+  @Getter getCreateResolutionStep!: CreateResolutionIF
+  @Getter getCreateResolutionResource!: CreateResolutionResourceIF
+  @Getter isBaseCompany!: boolean
+  @Getter isTypeCoop!: boolean
+
+  // Enum for template
+  readonly RouteNames = RouteNames
+
+  get resolutionDate (): string {
+    const result = this.getCreateResolutionStep.resolutionDate
+      ? this.dateStrToPacificDate(this.getCreateResolutionStep.resolutionDate, true)
+      : '(Not Entered)'
+    return result
+  }
+
+  get resolutionText (): string {
+    const result = this.getCreateResolutionStep.resolutionText
+      ? this.getCreateResolutionStep.resolutionText
+      : '(Not Entered)'
+    return result
+  }
+
+  get signingParty (): string {
+    const signingParty = this.getCreateResolutionStep.signingPerson
+    const result = signingParty.givenName && signingParty.familyName
+      ? `${signingParty.givenName.trim()} ${signingParty.additionalName.trim()} ${signingParty.familyName.trim()}`
+      : '(Not Entered)'
+    return result
+  }
+
+  get signingDate (): string {
+    const result = this.getCreateResolutionStep.signingDate
+      ? this.dateStrToPacificDate(this.getCreateResolutionStep.signingDate, true)
+      : '(Not Entered)'
+    return result
+  }
+
+  /** Is true when all validation items are invalid. */
+  get allResolutionValidationItemsInvalid (): boolean {
+    const validationItemsDetails = this.getCreateResolutionStep.validationDetail.validationItemDetails
+    const invalidValidationItems = validationItemsDetails.filter(item => item.valid === false)
+    const result = validationItemsDetails.length === invalidValidationItems.length
+    return result
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import '@/assets/styles/theme.scss';
+
+// styles specific to resolution summary section
+::v-deep #resolution-summary-section-3 {
+  // removes dotted line bottom border on text area
+  .v-text-field.v-input--is-readonly .v-input__slot:before {
+    border-style: None !important;
+  }
+
+  // remove text area padding
+  .v-text-field.v-text-field--enclosed:not(.v-text-field--rounded) > .v-input__control >
+  .v-input__slot, .v-text-field.v-text-field--enclosed .v-text-field__details {
+    padding: 0px !important;
+  }
+
+  #resolution-text {
+    margin-top: 0px !important;
+  }
+
+  .section-container {
+    padding-bottom: 2rem !important;
+  }
```

Comment:
It probably doesn't matter, but i don't think these classes need to be nested in the `v-deep`

Reply:
yeah i'll take a look. 👍

Reply:
V-deep is only needed if you're reaching into a sub-component.

Reply:
yeah i don't know why i did that.  oh yeah it was the text area specific styling.  maybe i can make it separate

Reply:
updated to apply `v-deep` only where req'd

![image](https://user-images.githubusercontent.com/6685223/143603123-32ae7c98-06d7-49e5-bec4-b2615e2cd0bc.png)